app/assets/stylesheets/common/core.scss

```diff
@@ -474,10 +474,30 @@ th.required::after {
   vertical-align: middle;
 }
 
+div.rendered-react-keyed-file-browser div.action-bar ul.item-actions {
+  overflow: auto;
+}
+
 div.rendered-react-keyed-file-browser div.action-bar input[type='search'] {
   width: 100%;
 }
 
+div.rendered-react-keyed-file-browser div.files table td.name div * {
+  padding-right: 0.5rem;
+}
+
+div.rendered-react-keyed-file-browser div.files tbody tr:hover {
+  background-color: $submission-row-hover;
+}
+
+.submission-download {
```

Comment:
In addition to the fill, add the rule `vertical-align: bottom;` (it'll make the vertical alignment for the icon a bit more centered with respect to the text)